✨ add blog post (#22)
GDG-Nantes/devfest2019: content/blog/accorde-tes-instruments.md

---
title: DevFest 2019 - Tune your instruments
brief: Tune your instruments
image: /images/blog/accorde-tes-instruments.jpg
date: 2019-06-14
draft: false
---

## CFP: Reminder

> As a reminder, you have until June 30 23:59 to propose a topic to #DevFestNantes!

Reviewer: ```suggestion
As a reminder, you have until June 30 23:59 to submit a talk to #DevFestNantes!
```

The Call for Paper is [here](https://conference-hall.io/public/event/o2j0dYZDa0W1Ils3kO0z)! 🎸 ["Thank you - Led Zeppelin"](https://www.youtube.com/watch?v=u1z4vkPWkLQ) 🎸

> Do not hesitate to consult our article: [the selection of the talks](https://medium.com/devfest-nantes/la-s%C3%A9lection-des-talks-842a622af11a) on our medium in order to know our way to select the DevFest talks.

Reviewer: ```suggestion
Do not hesitate to read our blogpost: [the selection of the talks](https://medium.com/devfest-nantes/la-s%C3%A9lection-des-talks-842a622af11a) in order to know the way we select the talks.
```

We also did a [dedicated meetup](https://www.meetup.com/en-GB/GDG-Nantes/events/261668797/) about last June 11th summary in one image [here] (https://twitter.com/gdgnantes/status/1138497945681170432).

> ## Inscriptions: save the date!

Reviewer: ```suggestion
## Tickets: save the date!
```

> As a reminder, we have opened a [lottery](https://www.billetweb.fr/devfest-Nantes-2019) which will allow you to buy a 2 day pass at the price of 99 € TTC if you are drawn (see [tweet](https://twitter.com/devfestnantes/status/1125388292986265600)).

Reviewer: ```suggestion
We have opened a [lottery](https://www.billetweb.fr/devfest-Nantes-2019) which will allow you to buy a 2-day pass at the price of 99 € TTC if you are drawn (see [tweet](https://twitter.com/devfestnantes/status/1125388292986265600)).
```

> 400 1-Day Pass will be on sale from 10 September 2019 at 10am! Do not forget to add it to your calendar 🎶 ["With or without you - U2"](https://www.youtube.com/watch?v=6DeDzsCGbsQ) 🎶

Reviewer: ```suggestion
400 1-Day Pass will be on sale the 10th September 2019 at 10am CEST! Do not forget to add it to your calendar 🎶 ["With or without you - U2"](https://www.youtube.com/watch?v=6DeDzsCGbsQ) 🎶
```
